core/src/modules/cloud.ts

```diff
@@ -0,0 +1,149 @@
+import os from "os";
+import path from "path";
+import tar from "tar";
+import fs from "fs";
+import FormData from "form-data";
+import { mkdtemp, copy, remove } from "fs-extra";
+import axios, { AxiosError, AxiosInstance } from "axios";
```

Comment:
using axios because isomorphic-fetch doesn't seem to play well with formData in node contexts (used when uploading files)

Reply:
That's a shame!  
... would it be terrible to remove isomprhic-fetch then?  Seems silly to have 2 request libraries...

Reply:
didn't want to remove isomorphic-fetch since it looks like some of these functions that use it may be called from the ui packages?

went back in to try to figure out why isomorphic-fetch wasn't working for me, and it looks like it was an issue with the typings. adding a `@ts-ignore` for it worked so I re-implemented the CloudClient with isomorphic-fetch and removed axios.